armi/physics/fuelPerformance/parameters.py

```diff
@@ -27,6 +27,13 @@ def _getFuelPerformanceBlockParams():
     pDefs = parameters.ParameterDefinitionCollection()
     with pDefs.createBuilder(default=0.0, location=ParamLocation.AVERAGE) as pb:
 
+        pb.defParam(
+            "fuelCladLocked",
+            units="",
+            default=False,
+            description="Boolean to indicate if the fuel is locked with the clad.",
```

Comment:
Can you add to this description a bit:

`Boolean to indicate if the fuel is locked with the clad. This is used to determine the expansion constraints for the fuel during thermal and/or burn-up expansion of the fuel and cladding materials.`

Reply:
https://github.com/terrapower/armi/pull/1038/commits/a04e0b8aede3e340699ca9b8d45159f369607af1